src/hdr_histogram.c

```diff
@@ -7,7 +7,9 @@
 #include <stdlib.h>
 #include <stdbool.h>
 #include <math.h>
+#ifdef __x86_64__
```

Comment:
If the issue can be solved just with an ifdef and no other changes, what's the point of even having the header at all? Why not just remove it?

Reply:
I suspect the header replaces some C functions with Intel intrinsics on x86, and I believe the code would run slower on Intel if we remove it.